test/HealthChecks.Azure.Data.Tables.Tests/AzureTableStorageHealthCheckTests.cs

```diff
@@ -55,36 +56,26 @@ public async Task return_healthy_when_only_checking_healthy_service()
     }
 
     [Fact]
-    public async Task return_healthy_when_checking_healthy_service_and_table()
+    public async Task return_healthy_when_checking_healthy_service_table()
     {
         using var tokenSource = new CancellationTokenSource();
 
-        _tableServiceClient
```

Comment:
note to other reviewers: we already have a test that is testing the code path where `_tableServiceClient` is being used: 

https://github.com/Xabaril/AspNetCore.Diagnostics.HealthChecks/blob/f587a637a1dc9850cce87d441ba173d50679aefc/test/HealthChecks.Azure.Data.Tables.Tests/TableServiceHealthCheckTests.cs#L36-L55

(this change looks like we test only 1 out of 2, but we keep testing both)